pages/linux/update-rc.d.md

```diff
@@ -1,19 +1,19 @@
 # update-rc.d
 
-> Install and remove System-V style init script links.
+> Install and remove service.
 
 - Install a service:
 
-`update-rc.d {{name}} defaults`
+`update-rc.d mysql defaults`
```

Comment:
Please put mysql in tokens(`{}`). And mention that this file should belong in `/etc/init.d/` folder at the top. (Preferably as a second heading line.)